Analysis/Tasks/PWGHF/CMakeLists.txt

```diff
@@ -38,6 +38,11 @@ o2_add_dpl_workflow(hf-lc-candidate-selector
                     PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore O2::DetectorsVertexing
                     COMPONENT_NAME Analysis)
 
+o2_add_dpl_workflow(hf-jpsi-candidate-selector
+					SOURCES HFJpsiCandidateSelector.cxx
+					PUBLIC_LINK_LIBRARIES O2::Framework O2::AnalysisDataModel O2::AnalysisCore O2::DetectorsVertexing
+					COMPONENT_NAME Analysis)
```

Comment:
Do not use tabs anywhere in the code. Use spaces only.
Please check the rest of your changes too.